src/controller/config.js

```diff
@@ -24,9 +24,10 @@ let currCoordinates = null
 window.addEventListener('load', (e) => {
     setNoImage()
     let editBodyPartList = window.api.getEditBodyPart()
-    let editBodyPart = editBodyPartList[0]
-    bodyPartId = editBodyPartList[1]
-    if(editBodyPart){
+    if(editBodyPartList){
+        let editBodyPart = editBodyPartList[0]
+        bodyPartId = editBodyPartList[1]
+        window.api.clearEditBodyPart()
```

Comment:
Once we have the body part we can clear it out of local storage so it is not loaded every time you open this page